galley/pkg/util/worker.go

```diff
@@ -107,6 +107,7 @@ func (w *Worker) Stop() {
 		w.stateLock.Unlock()
 		return
 	}
+	w.started = false
```

Comment:
why are you unsetting started here? Are you re-running workers somewhere?

Reply:
This was accidentally added. Removed. The watchers/sources can be started/stopped multiple times, so this was not compatible with that.